build/configs/rtl8720e/hello/defconfig

```diff
@@ -142,7 +142,7 @@ CONFIG_INIC_IPC_HIGH_TP=1
 # Realtek RTL8720E BLE Support
 #
 CONFIG_AMEBALITE_BLE=y
-CONFIG_FTL_ENABLED=y
+# CONFIG_FTL_ENABLED=y
```

Comment:
Please don't update this manually. After updating .config through the menuconfig, please copy .config into defconfig.

Reply:
Thanks for your help, I have fixed it as you said.